### PR TITLE
Make clean was not unlinking pipeline-editor

### LINKED ISSUE #1814: Make clean does not unlink pipeline editor (from make dev-link)
After a make clean install and reload of the lab application, I still see console.logs from a dev-linked pipeline editor.

### PR DESCRIPTION
**Pull request by ajbozarth**
`make clean` was trying to uninstall the dev instance of pipeline-editor instead of unlink.

Fixes #1814

### How was this pull request tested?

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,9 +63,8 @@ uninstall:
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/pipeline-editor-extension)
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/python-editor-extension)
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/r-editor-extension)
-	- jupyter labextension uninstall @elyra/pipeline-services
-	- jupyter labextension uninstall @elyra/pipeline-editor
-	pip uninstall -y jupyterlab-git
```
Comment (karlaspuldaro) [inline Makefile:68]: why are we removing git uninstall?

Comment (ajbozarth) [reply Makefile:68]: that was a typo, thanks

**Review by karlaspuldaro — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -63,8 +63,8 @@ uninstall:
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/pipeline-editor-extension)
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/python-editor-extension)
 	$(call UNINSTALL_LAB_EXTENSION,@elyra/r-editor-extension)
-	- jupyter labextension uninstall @elyra/pipeline-services
-	- jupyter labextension uninstall @elyra/pipeline-editor
+	- jupyter labextension unlink @elyra/pipeline-services
+	- jupyter labextension unlink @elyra/pipeline-editor
```
Comment (lresende) [inline Makefile:67]: will this still work with you don't have a linked extension?

Comment (ajbozarth) [reply Makefile:67]: that's what the `-` is for, it tells make to ignore failure for that line and continue on.

**Pull request merged**